tox.ini

```diff
@@ -10,7 +10,7 @@ commands =
 deps =
     nose
     psycopg2
-    MySQL-python
+    mysqlclient
```

Comment:
we were previously using 'mysqlclient' for some python versions (3.x) using direct git repository URL (see below), and for 2.x using the old MySQL-python package. Switched to just use the 'mysqlclient' fork (which is same as the one pointed to via the GitHub URL below) across the board - it seems very actively maintained and compliant with both python 2.x and 3.x - https://github.com/PyMySQL/mysqlclient-python